### DISCUSSION TITLE
install .img files

### DISCUSSION THREAD

**Discussion by azatecas** [utmapp/UTM#3391] [UTM Help (macOS)]
hello, I have an m1 mac, I would like to install raspberry pi made operating system such as Pop!_OS 21.10 that is made specifically for arm. when i download it it only comes as a .img file type instead on a .iso.  i havent had any luck. i tried converting the .img file into .iso but the vm didn't boot. any tips for installing .img files on utm would be helpful

**Comment by Andrupka** [comment]
It's an IMG so create a non-removable drive, then use the IMG as a hard drive image, not as CD image...

**Reply by azatecas** [reply]
Thank you for the quick response. this is what i've done based on your suggestion:
![Screen Shot 2021-12-17 at 9 32 16 AM](https://user-images.githubusercontent.com/29685279/146560185-dd701080-beb1-43ea-8a00-ea294e9f588f.png)
![Screen Shot 2021-12-17 at 9 32 44 AM](https://user-images.githubusercontent.com/29685279/146560230-54fc9433-486d-4331-9cc3-c0297bcb7569.png)
![Screen Shot 2021-12-17 at 9 33 34 AM](https://user-images.githubusercontent.com/29685279/146560248-55bf8c51-58fb-4954-990c-de8775605e90.png)


I get stuck on the Shell Screen. Did i do the suggestion correctly or is there something im not seeing? im very new at UTM/QEMU

**Reply by azatecas** [reply]
I have also tried importing the IMG file instead of selecting it from removable storage but same resul
![Screen Shot 2021-12-17 at 9 52 50 AM](https://user-images.githubusercontent.com/29685279/146562999-170e99ce-9d65-4fd0-be38-8cd5eb80f1f2.png)
t

**Reply by chrisguida** [reply]
I would also like to know the answer to this!

**Reply by g-cqd** [reply]
Any follow-up on this thing ?

**Reply by anzbert** [reply]
i am stuck on the same problem

**Reply by g-cqd** [reply]
I tried many things but I don't see how it should be done. I tried the `manjaro-arm-installer` script on a USB drive and wrote an image from that key and tried it that way and I couldn't boot on it.

I tried to mount it as a classic drive or virtIO one and still was not able to boot that thing.

It's incredible I succeed to emulate a chrome os flex from a raw binary intercepted through the ChromeOS Recovery Tool extension execution but still not being able to boot a dedicated img.

**Reply by BananaCakeComputer** [reply]
> It's an IMG so create a non-removable drive, then use the IMG as a hard drive image, not as CD image...

Why a non removable for floppy?

**Reply by g-cqd** [reply]
> > It's an IMG so create a non-removable drive, then use the IMG as a hard drive image, not as CD image...
> 
> Why a non removable for floppy?

So many questions unanswered...

**Reply by g-cqd** [reply]
I found ! img files can be converted to vhdx files with qemu-img convert. Then you take the path of virtualizing windows, check importing a vhdx, uncheck installing windows, select your freshly converted vhdx and you're good to go ;)

**Reply by g-cqd** [reply]
the only way i succeed to make it work then is to replace the display with a serial interface set to the built-in terminal. it may be because i use a minimal manjaro img.
